e2e/transactions/transactions_test.go

```diff
@@ -764,30 +783,43 @@ func (s *TransactionsTestSuite) TestEvictionOfQueuedTransactions() {
 	s.NoError(s.Backend.AccountManager().SelectAccount(TestConfig.Account1.Address, TestConfig.Account1.Password))
 
 	txQueue := s.Backend.TxQueueManager().TransactionQueue()
-	var i = 0
+
+	var i = int32(0)
 	txIDs := [txqueue.DefaultTxQueueCap + 5 + 10]common.QueuedTxID{}
 	s.Backend.TxQueueManager().SetTransactionQueueHandler(func(queuedTx *common.QueuedTx) {
-		log.Info("tx enqueued", "i", i+1, "queue size", txQueue.Count(), "id", queuedTx.ID)
-		txIDs[i] = queuedTx.ID
-		i++
+		n := atomic.LoadInt32(&i)
+		log.Info("tx enqueued", "i", n+1, "queue size", txQueue.Count(), "id", queuedTx.ID())
+		txIDs[n] = queuedTx.ID()
+
+		atomic.AddInt32(&i, 1)
 	})
 
 	s.Zero(txQueue.Count(), "transaction count should be zero")
 
+	wg := sync.WaitGroup{}
```

Comment:
Same here. I have to admit having seen the `var` notation for it more often during my go years.

Reply:
ok

Reply:
ok